tests/overlay/test

```diff
@@ -943,7 +943,6 @@ sub context_test {
     print "=====================================================\n";
     print "Bad Entrypoint tests.\n";
     print "=====================================================\n";
-    test_90_1();
     test_90_2();
```

Comment:
Does it make sense to retain any entrypoint tests (i.e. test_90_2)?
And should we instead be adding tests which confirm that entrypoint is properly denied on the overlay inode when we don't allow entrypoint to that type?

Reply:
I guess retaining entrypoint test_90_2 does not hurt. 

For testing test_90_1() properly, so we will have to come up with a new label which denies entrypoint and mount using that label and then run test_90_1. 

@rhatdan , do you want to take a shot at that and rewrite test_90_1() for context mounts.

Reply:
ro_t will deny execute permission to overlay_client_t. So it will not be a entrypoint test. We will need to define a new label where exec is allowed to client but entrypoint is not.